Resources/translations/sulu/backend.fr.xlf

```diff
@@ -98,51 +98,51 @@
             </trans-unit>
             <trans-unit id="25">
                 <source>sulu_form.type.text</source>
-                <target>Simple Textfield</target>
+                <target>Texte simple</target>
             </trans-unit>
             <trans-unit id="26">
                 <source>sulu_form.type.textarea</source>
-                <target>Multiline Textfield</target>
+                <target>Texte multiligne</target>
             </trans-unit>
             <trans-unit id="27">
                 <source>sulu_form.type.date</source>
                 <target>Date</target>
             </trans-unit>
             <trans-unit id="28">
                 <source>sulu_form.type.headline</source>
-                <target>Gros titre</target>
+                <target>-Manchette-</target>
```

Comment:
please remove the `-` in the new 0.4 version they types are grouped to avoid conflicts for the content manager.